Game/multiplayer-project/Assets/Scripts/Http/Models/input_data.json

```diff
@@ -0,0 +1,36 @@
+{
+    "general_knowledge":{
+        "items":{
+            "plants":["glowy_mushroom","medicinal_herb"]
+        },
+        "action":["harvest","inspect","stash","attack","heal","moveto"],
+        "players":{
+            "0":{
+                "temperament":"greedy"
```

Comment:
we should be careful not to go to granular, but in other hand it is great to provides such information to LLM
-> let's keep it simple and iterate